certificate_agent.py

```diff
@@ -120,11 +120,12 @@ def main():
                     name=name,
                     course_id=course_id,
                     grade=grade,
+                    designation=designation,
```

Comment:
We're passing in designation to `format`, but I'm not seeing `designation` anywhere in the formatted string.